marketplaceApp/app/controllers/items_controller.rb

```diff
@@ -1,13 +1,13 @@
 class ItemsController < ApplicationController
   def index
-    #order = "name"
-    #@items = Item.sorted_by(order)
-    @items = Item.all
+    order = params[:order] || "name"
+    @items = Item.all.order(order)
   end
     
   def show
-    id = params[:id]
-    @item = Item.find(id)
+    if params[:id]
+      @item = Item.find(params[:id])
+      @seller = User.find(@item.user_id).name
```

Comment:
I think every user is both a seller and a buyer, so should this be @user? I'm not sure how we want to pass these into the view from the controller

Reply:
yeah. i think this is probably the easiest way to pass it into the view from the controller with the instance variable.
the code that was merged yesterday to master doesn't have any rspec tests. do you guys wanna write those first before i write them for this PR?

Reply:
yeah good point.